oeps/conf.py

```diff
@@ -81,7 +81,7 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = 'en'
```

Comment:
Passing default language as `None` generates warning and is automatically updated to `en`. See [CHANGELOG](https://github.com/sphinx-doc/sphinx/blob/5.x/CHANGES#L80) for details.